.github/workflow-scripts/actOnLabel.js

```diff
@@ -131,7 +131,9 @@ module.exports = async (github, context, labelWithContext) => {
           `| --- | --- |\n` +
           `| :information_source: | It looks like your issue or the example you provided uses an [unsupported version of React Native](https://github.com/reactwg/react-native-releases/blob/main/README.md#releases-support-policy). Due to the number of issues we receive, we're currently only accepting new issues against one of the supported versions. Please [upgrade](https://reactnative.dev/docs/upgrading) to latest and verify if the issue persists (alternatively, create a new project and repro the issue in it). If you cannot upgrade, please open your issue on [StackOverflow](https://stackoverflow.com/questions/tagged/react-native) to get further community support. |`,
       );
-      await requestAuthorFeedback();
+      /* NOTE: This will close the issue even if the
+      version might be supported but is not mentioned in the proper format. */
+      await closeIssue();
```

Comment:
This is the simplest way to do this but we know that there are some serious :) concerns - that I have also mentioned in the multiline comments here :)

Reply:
Yeah totally. Plus I would try to be a bit less agressive. I.e. 5 versions ago: autoclose. 3 versions ago: just apply the label

Reply:
> I.e. 5 versions ago: autoclose. 3 versions ago: just apply the label

💯 🙌

Reply:
yeah I agree with @cortinico - let's have a "grace window" of maybe the first 2/3 minors in the unsupported window (so for example, right now those would be 69/68/67) and anything lower than that, let's insta-close